builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -531,14 +531,15 @@ def writeServerlessFile(config, environment_logical_id){
 	sh "sed -i -- 's/\${file(deployment-env.yml):providerTimeout}/${config['providerTimeout']}/g' serverless.yml"
 	sh "sed -i -- 's/\${file(deployment-env.yml):securityGroupIds}/${config['securityGroupIds']}/g' serverless.yml"
 	sh "sed -i -- 's/\${file(deployment-env.yml):subnetIds}/${config['subnetIds'] }/g' serverless.yml"
+  sh "sed -i -- '/#Start:resources/,/#End:resources/d' ./serverless.yml"
+  sh "sed -i -- '/#Start:isScheduleEnabled/,/#End:isScheduleEnabled/d' ./serverless.yml"
```

Comment:
How about other sections like `isDynamoDbEnabled`, `isSQSEventEnabled` etc. (actually everything under `eventsDisabled`)? Those are not required in API build workflows. Why are we only removing `isScheduleEnabled` section?